app/components/joyride-foundation.js

```diff
@@ -43,6 +54,10 @@ export default Ember.Component.extend({
         function showJoyrideOverlay(){
           Ember.$(".joyride-view-background").show();
         }
+
+        function setSeenTour(){
+          _this.set("session.seenTour", _this.get("session.seenTour") + "," + _this.get("currentUserId"));
```

Comment:
Changes done as per the suggestion https://github.com/crossroads/app.goodcity/commit/9dd52fc1b01e259945f1c51e4fa70185da8b0ac0
cc: @steveyken